Cache/Main.lean

```diff
@@ -43,12 +43,16 @@ Which will download the cache for:
 * Every Lean file inside 'Mathlib/Data/'
 * Everything that's needed for the above"
 
-open System (FilePath) in
+open System in
 /-- Note that this normalizes the path strings, which is needed when running from a unix shell
 (which uses `/` in paths) on windows (which uses `\` in paths) as otherwise our filename keys won't
 match. -/
 def toPaths (args : List String) : List FilePath :=
-  args.map (FilePath.mk · |>.normalize)
+  args.map fun arg =>
+    if arg.endsWith ".lean" then
+      FilePath.mk arg |>.normalize
+    else
+      mkFilePath (arg.splitOn ".") |>.withExtension "lean"
```

Comment:
I guess there is a small risk that theoretically this doesn't work with imports where there is a `.` that is escaped by french quotes?
Should that be mentioned in a comment?

Can we alternatively invoke some existing Lean function that turns module names into filepaths?

Reply:
There's not a single step, but I've replaced it with 
```
mkFilePath (arg.toName.components.map Name.toString) |>.withExtension "lean"
```
which handles French quotes.

Reply:
Thanks, LGTM